homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -143,7 +131,7 @@
             " as the drive train battery won't connect."
             " Don't set the intervals too low.")
 
-        data_store = LeafDataStore(leaf, hass, car_config)
+        data_store = LeafDataStore(hass, leaf, car_config)
         hass.data[DATA_LEAF][leaf.vin] = data_store
 
         for component in LEAF_COMPONENTS:
```

Comment:
Why do we not load the device tracker platform in this loop?

Reply:
Is the device tracker platform only supported for cars with nconnect?

Reply:
Yeah, I never developed it out when I built the initial codebase as I have a CarWings-based car so couldn't test it. Don't know what @filcole drives

Reply:
I believe so.  I have a 30kWh 65 UK plate Nissan Leaf which provides device tracker information.  I believe older 24kWh Leafs and enV200 (electric mini-vans) don't provide tracking information.  @BenWoodford if you still have 24kWh leaf would love help wit testing.  I will message you.

Reply:
Hi, my 24kWh does NOT support tracking.